src/css/blocks/product-card.css

```diff
@@ -0,0 +1,98 @@
+:root {
+  --main-accent: #060b35;
+  --blue-accent: #00acdc;
+  --secondary-accent: #616070;
+
+  /* --font-size-title: 12px;
+  --font-size-product-number: 10px;
+  --font-size-feedback: 10px;
+  --font-size-price-descr: 12px;
+  --font-size-price-value: 16px;
+  --font-size-button: 14px; */
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.product-card__title {
+  margin-top: 40px;
+  color: var(--main-accent);
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.product-card__product-number {
+  margin-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-accent);
+}
+
+.product-card__rating-wrap {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+  
+.product-card__feedback {
+  font-size: 10px;
+  cursor: pointer;
+  color: var(--main-accent);
+}
+
+.product-card__price-wrap {
+  margin-top: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 18px;
+}
+
+.product-card__price-descr {
+  color: var(--secondary-accent);
+  font-size: 12px;
+}
+
+.product-card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--main-accent);
+}
+
+.product-card__button {
+  margin-top: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  background-color: var(--blue-accent);
+  color: #fff;
+  border-radius: 5px;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.product-card__button:hover {
+  background: #fff;
+  color: var(--blue-accent);
+  border: 1px solid var(--blue-accent);
```

Comment:
According to checklist:
[STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover